src/Microsoft.Sbom.Api/Config/Args/GenerationArgs.cs

```diff
@@ -116,6 +116,15 @@ public class GenerationArgs : GenerationAndValidationCommonArgs
     [ArgDescription("If set to true, we will attempt to fetch license information of packages detected in the SBOM from the ClearlyDefinedApi.")]
     public bool? FetchLicenseInformation { get; set; }
 
+    /// <summary>
+    /// Specifies the timeout in seconds for fetching the license information. Defaults to 30 seconds. Has no effect if
+    /// FetchLicenseInformation (li) argument is false or not provided.
+    /// </summary>
+    [ArgShortcut("lto")]
+    [ArgDescription("Specifies the timeout in seconds for fetching the license information. Defaults to 30 seconds. " +
+        "Has no effect if the FetchLicenseInformation (li) argument is false or not provided. ")]
+    public int? LicenseInformationTimeout { get; set; }
```

Comment:
We should specify units in the name to avoid confusion. Maybe 
```suggestion
    public int? LicenseInformationTimeoutInSeconds { get; set; }
```

Reply:
I agree, I've updated all the timeout variable names to specify InSeconds